### PR TITLE
feat: Add bill icon

### PR DESCRIPTION
**Pull request by paultranvan**
This icon will be useful for the document qualification

### CODE REVIEW

**Review by ptbrowne — APPROVED** [review]
Thanks for the doc !

File: react/Icon/Readme.md [hunk 1]
```diff
@@ -600,3 +602,12 @@ const handleInputRangeChange = ev => {
...
 ```
+
+## Add an Icon
+
+If you want to add a new icon, you must follow these steps:
+
+* Add the SVG in the `assets/icons` folder
+* Generate the react component, simply by running `scripts/generate-svgr-icon.sh`
+* Update this README by adding your icon in the list
+* Update the tests by running `yarn test --updateSnapshot`
```
Comment (ptbrowne) [inline react/Icon/Readme.md:613]: This should be in the README, not in the example of the styleguide.

**Review by ptbrowne — CHANGES_REQUESTED** [review]
Can you move the "Add an icon" doc to the readme ? Here you are adding it to the styleguide.

File: react/Icon/Readme.md (hunk 1 above)
Comment (paultranvan) [reply react/Icon/Readme.md:613]: Well, this is a readme :smile: But sure, I'll move it to the main one

Comment (paultranvan) [reply react/Icon/Readme.md]: Done with e04f839

**Review by ptbrowne — APPROVED** [review]
Thanks

**Pull request merged**